.vsts-ci.yml

```diff
@@ -1,26 +1,48 @@
+resources:
+  repositories:
+  # shared library repository
+  - repository: arcade
+    type: github
+    endpoint: jofortes-personal
+    name: dotnet/arcade
+    ref: refs/heads/master
+
+variables:
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+
 phases:
-- phase: First_phase
-  queue:
-    name: Hosted VS2017
-  steps:
-  - script: echo Hello World!
+- template: /eng/common/templates/phases/base.yml@arcade
```

Comment:
When we do this, we should used the template that is checked in the repo. Dependency flow should update it if needed.